mobile_verifier/src/cli/server.rs

```diff
@@ -101,10 +104,63 @@ impl Cmd {
             settings.usa_and_mexico_fencing_resolution()?,
         )?;
 
+        let (oracle_boosting_reports, oracle_boosting_reports_server) =
+            file_sink::FileSinkBuilder::new(
+                FileType::OracleBoostingReport,
+                store_base_path,
+                file_upload.clone(),
+                concat!(env!("CARGO_PKG_NAME"), "_oracle_boosting_report"),
+            )
+            .auto_commit(false)
+            .roll_time(Duration::minutes(15))
+            .create()
+            .await?;
+
+        let urbanization: Urbanization<DiskTreeMap, _> = Urbanization::new(usa_geofence.clone());
+        let footfall: Footfall<DiskTreeMap> = Footfall::new();
+        let landtype: Landtype<DiskTreeMap> = Landtype::new();
+        let hex_boost_data = HexBoostData::new(footfall, landtype, urbanization);
+
+        // Data sets and downloaders
+        let data_sets_file_store = FileStore::from_settings(&settings.data_sets).await?;
+        let footfall_data_set_downloader = DataSetDownloaderDaemon::new(
```

Comment:
Might want to follow the new convention used to setup new daemons as part of https://github.com/helium/oracles/pull/796